avocado/core/nrunner/runner.py

```diff
@@ -43,6 +43,10 @@ def check_runnables_runner_requirements(runnables, runners_registry=None):
 
 class BaseRunner(RunnableRunner):
 
+    #: The "main Avocado" configuration keys (AKA namespaces) that
+    #: this runners makes use of.
+    CONFIGURATION_USED = []
```

Comment:
This is an important change to the runners and adds a new task to the runners development. IMO we need to update our documentation about [writing new runner scripts](https://avocado-framework.readthedocs.io/en/latest/guides/contributor/chapters/runners.html#writing-new-runner-scripts)

Reply:
Because that is set on the base class, new runners don't *need* that set it.  In fact, only three of the runners currently set it:

```
asset.py:    CONFIGURATION_USED = ['datadir.paths.cache_dirs']
avocado_instrumented.py:    CONFIGURATION_USED = ['run.test_parameters',
exec_test.py:    CONFIGURATION_USED = ['run.test_parameters',
```

I'm only halfway convinced that we should add more than the basic information in a tutorial like documentation.  New developers would probably copy and paste an empty assignment without the need to do so.

Let me know if I'm missing something and I can certainly get to fully convinced status :).

Reply:
Also, I forgot to mention that, `sysinfo` runner also uses [config options](https://github.com/avocado-framework/avocado/blob/master/avocado/core/runners/sysinfo.py#L38). Which should be changed here as well.